sci-libs/openblas/openblas-0.2.20.ebuild

```diff
@@ -21,8 +21,6 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 MULTILIB_WRAPPED_HEADERS=(
-	/usr/include/openblas/cblas.h
-	/usr/include/openblas/f77blas.h
 	/usr/include/openblas/openblas_config.h
```

Comment:
Thanks, that's cleaner!  Could you please add a reference in the commit message to the solution of header problem in bugs.g.o?